mlflow/R/mlflow/R/model-h2o.R

```diff
@@ -31,13 +31,8 @@ mlflow_save_model.H2OModel <- function(model,
   )
   yaml::write_yaml(settings, file.path(model_data_path, "h2o.yaml"))
 
-  # needed because R and python packages may not have the same patch number
```

Comment:
Removed this because setting the versions equal requires the whole version number, but `remove_patch_version` only returned `3.30`.